tools/unit_tests/test_rtg.py

```diff
@@ -53,7 +53,7 @@ def plotter_multiple_graphs():
         print("Multiple Graphs: Should be showing multiple graphs.")
         try:
             test_object.run()
-        except Exception as e:
+        except Exception:
```

Comment:
Can you iron out what errors might pop during runtime? I would prefer that to `Exception`